wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandlerTests.swift

```diff
@@ -352,4 +353,90 @@ final class CreateGroupConversationActionHandlerTests: ActionHandlerTestBase<Cre
         ])
     }
 
+    func testThatItCallsResultHandler_OnUnreachableDomainsError() {
+        syncMOC.performGroupedAndWait { [self] _ in
+            // Given
+            BackendInfo.apiVersion = .v4
+            let unreachableDomain = "foma.wire.link"
+            let unreachableUser = ZMUser.insertNewObject(in: self.syncMOC)
+            unreachableUser.remoteIdentifier = UUID()
+            unreachableUser.domain = unreachableDomain
```

Comment:
I don't see the `unreachableUser` being used anywhere in the test, is it needed?

Reply:
no, it's not needed